coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
Yes all the json files will be stored in `data` directory. It's something like -
```
data
   files.json
   coala.json
   file_data.json
   .....
```

Knowing the parent directory I can access the files as per the need.